src/client/src/components/pages/courses/addCourses/index.jsx

```diff
@@ -71,16 +73,19 @@ export default class index extends Component {
     const fields = { ...this.state };
     const error = validationForm(fields);
     if (error) return this.setState({ error });
-
     this.addCourse(fields);
   };
 
   render() {
+    const { type } = this.state;
+    const fieldsSetClone = [...fieldSet]
```

Comment:
Cloned Array ,, If you ask why i clone it  because if we use fieldSet each time we switch from pastoral to training the second row of the original array will be removed  ( i think that trouble )